Adds note on testing with deferLoading() (#5745)
laravel-filament/filament: packages/tables/docs/07-testing.md

> Note that if your table uses pagination, `assertCanSeeTableRecords()` will only check for records on the first page. To switch page, call `set('page', 2)`.

> > Note that if your table uses `deferLoading()`, you should call `loadTable()` before `assertCanSeeTableRecords()` to render the table records directly.

Reviewer: ```suggestion
> Note that if your table uses `deferLoading()`, you should call `loadTable()` before `assertCanSeeTableRecords()`.
```

> > Note that if your table uses deferLoading(), you must call `set('isTableLoaded', true)` before `assertCanSeeTableRecords()` to force the the component to render the records.

Reviewer: ```suggestion
> Note that if your table uses `deferLoading()`, you should call `set('isTableLoaded', true)` before `assertCanSeeTableRecords()` to render the table records directly.
```
